var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -29,43 +29,56 @@
 from spack import *
 
 
-class Rose(Package):
+class Rose(AutotoolsPackage):
     """A compiler infrastructure to build source-to-source program
        transformation and analysis tools.
        (Developed at Lawrence Livermore National Lab)"""
 
     homepage = "http://rosecompiler.org/"
-    url = "https://github.com/rose-compiler/rose/archive/v0.9.7.tar.gz"
 
-    version('0.9.7', 'e14ce5250078df4b09f4f40559d46c75')
-    version('master', branch='master',
+    version('develop', branch='master',
+            git='https://github.com/rose-compiler/rose-develop.git')
+    version('0.9.7.0', commit='992c21ad06893bc1e9e7688afe0562eee0fda021',
             git='https://github.com/rose-compiler/rose.git')
-
-    patch('add_spack_compiler_recognition.patch')
+    version('0.9.9.0', commit='14d3ebdd7f83cbcc295e6ed45b45d2e9ed32b5ff',
+            git='https://github.com/rose-compiler/rose.git')
+    version('0.9.9.83', commit='1742d1773da7525378f15e40dfa391a3a7ac67df',
+            git='rose-dev@rosecompiler1.llnl.gov:rose/scratch/rose.git')
```

Comment:
I have no access to that place...

Reply:
I've cleaned this up in an updated commit. Only `0.9.7.0` and `0.9.9.0` should be present, both from the `rose-compiler/rose-develop` repository.